frontend/src/routes/Applications/components/TimeWindowLabels.tsx

```diff
@@ -44,17 +46,17 @@ export function TimeWindowLabels(props: ITimeWindowLabelsProps) {
                 <div className="timeWindow-labels-popover-content">
                     {missingData ? (
                         <div className="timeWindow-content">
-                            'Go to the Editor tab to view time window information.'
+                            {t('Go to the Editor tab to view time window information.')}
                         </div>
                     ) : (
                         <Fragment>
-                            <div className="timeWindow-title">Days of the week</div>
+                            <div className="timeWindow-title">{t('Days of the week')}</div>
```

Comment:
Will these day names be translated?

Reply:
ah yes, I tried figuring that one out but reverted to push up in progress work.  I slacked a question on how to pass the dynamic value from the spec. from :
```
    <TimeWindowLabels
    subName={sub.metadata.name as string}
    type={sub.spec.timewindow?.windowtype}
    days={sub.spec.timewindow?.daysofweek}
    timezone={sub.spec.timewindow?.location}
    ranges={sub.spec.timewindow?.hours}
    missingData={sub.spec.timewindow?.missingData}
    />
```

 so will work on this now